waspc/cli/src/Wasp/Cli/Command/Telemetry/Project.hs

```diff
@@ -37,6 +38,7 @@ considerSendingData telemetryCacheDirPath userSignature projectHash cmdCall = do
 
   let relevantLastCheckIn = case cmdCall of
         Command.Call.Build -> _lastCheckInBuild projectCache
+        Command.Call.Deploy _ -> _lastCheckInDeploy projectCache
         _ -> _lastCheckIn projectCache
```

Comment:
Hm yeah `_` left of `->` in case can be tricky in that way, because it captures everything, but what is the alternative, listing everything? That is kind of yucky! Although I guess better to have boilterplate than a runtime error that could have been caught in compile time, but still.

Reply:
Ah ok, yeah sorry missed the question there too. I just noticed it but yeah no compiler help. The downside to listing them all here would be any new command call people would have to update Telemetry too. Unsure if that is worth it? I guess it depends on if they would require special last check treatment (I would guess most would not, so `_` may be preferrable?).

Reply:
Yeah I am also not sure, I would probably leave `_` as it is, because as you said most won't care about this.